src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/tests/PageList.test.js

```diff
@@ -90,8 +90,6 @@ beforeEach(() => {
 
 test('Render PageList', () => {
     const webspaceKey = observable.box('sulu');
-    // $FlowFixMe
```

Comment:
Resp. does this not work anymore? Although I find the solution with the `defaultWebspace` cleaner anyway, I am just curious how that changed.

Reply:
the `$FlowFixMe`comment now needs to be at the "primary location": https://medium.com/flow-type/making-flow-error-suppressions-more-specific-280aa4e3c95c

i guess in our case this would be the line above the `<PageList route ...` JSX inside of the `mount()` call. but have not tested this to be honest

Reply:
Maybe that would be the better way to solve it instead of casting everything to any? :thinking:

Reply:
yes, you are right - changed it to not cast to `any`